kong/router/atc.lua

```diff
@@ -553,14 +642,29 @@ function _M:exec(ctx)
     headers_key = get_headers_key(headers)
   end
 
+  local queries, queries_key
+  if self.match_queries then
+    local err
+    queries, err = get_uri_args()
+    if err == "truncated" then
+      local lua_max_uri_args = kong and kong.configuration and kong.configuration.lua_max_uri_args or 100
```

Comment:
This needs to be added to `‎kong/conf_loader/init.lua`, `kong_defaults.lua` and `kong.conf.default.` The same way as `lua_max_req_headers`.

Reply:
We already have this option in `kong/conf_loader/init.lua` and others.

Reply:
Didn't knew that. Then we are good here.